nsca_client/README.rst

```diff
@@ -59,32 +69,25 @@ E.g:
         def nsca_check_mails(self):
             mails = self.search([('state', '=', 'exception')])
             if mails:
-                return (1, u"%s mails not sent" % len(mails))
-            return (0, u"OK")
+                return (1, u"%s mails not sent" % len(mails), {
+                  'exceptions': {'value': mails}})
+            return (0, u"OK", {'exceptions': {'value': mails}})
```

Comment:
The README does not mention the `exceptions` key, is it required? If so, what are the others keys?

Reply:
It is the label of the performance data sent. I will upgrade the README.